tests/test_stubs.py

```diff
@@ -52,7 +56,12 @@ def test_fail(path, py2_arg):
         lines = fin.readlines()
 
     errors = defaultdict(lambda: "")
-    for error_line in stdout.split("\n"):
+    error_lines = stdout.rstrip("\n").split("\n")
+    assert re.match(
```

Comment:
Now mypy always outputs at least one summary line; check that this line is as expected (in case it goes away in a later version) and then drop it.